chrome/host.json

```diff
@@ -4,6 +4,6 @@
   "path": "%%replace%%",
   "type": "stdio",
   "allowed_origins": [
-    "chrome-extension://jegbgfamcgeocbfeebacnkociplhmfbk/"
+    "chrome-extension://gkndoddhagggkjfkojkmnimeafhmpeod/"
```

Comment:
Wait, why did I change this? I think this shouldn't have been included in the PR, sorry.